release-1.8/release_team.md

```diff
@@ -0,0 +1,12 @@
+|  **Role** | **Name** (**GitHub/Slack ID**)  | **Shadow Name(s) (GitHub/Slack ID), ...**
+|  ------ | ------ | ------ |
+|  Lead | | |
+|  Secondary | | |
+|  Features || |
+|  Branch Manager | | |
+|  Test Infra | | | |
+|  Docs || |
+|  Bugs | | | |
+|  CI Signal | | | |
```

Comment:
+1 for having roles start immediately

My only concern with merging would be spreading leads thin.